catalog/nsili/catalog-nsili-sourcestoquery-ui/src/main/webapp/templates/sourcesToQueryTable.handlebars

```diff
@@ -0,0 +1,35 @@
+{{!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ --}}
+
+<table class="table table-striped  table-hover table-condensed">
+    <thead>
+        <tr>
+            <th>Source Id</th>
+            <th class="right"><a href="#" class="fa fa-plus-square fa-lg plus-button showCreateModal" style="text-decoration:none;"></th>
```

Comment:
I think we should try to avoid inline styling if we can.  Recommend making a css class for `text-decoration:none;`